core/trino-main/src/test/java/io/trino/connector/MockConnector.java

```diff
@@ -518,6 +518,9 @@ public void setColumnComment(ConnectorSession session, ConnectorTableHandle tabl
         @Override
         public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnMetadata column) {}
 
+        @Override
+        public void setColumnType(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column, Type type) {}
```

Comment:
Should we have some mockconnector-based tests?

(instead of testing with mongo connector)

Reply:
I thought the existing `TestSetColumnTypeTask` using `MockMetadata` was sufficient. Please let me know if you have specific recommendations that `MockConnector` should use.